packages/polling-controller/src/StaticIntervalPollingController.ts

```diff
@@ -0,0 +1,75 @@
+import { BaseController, BaseControllerV1 } from '@metamask/base-controller';
+import type { NetworkClientId } from '@metamask/network-controller';
+import type { Json } from '@metamask/utils';
+
+import {
+  PollingControllerBaseMixin,
+  getKey,
+} from './PollingController-abstract';
+import type { PollingTokenSetId } from './PollingController-abstract';
+
+type Constructor = new (...args: any[]) => object;
+
+/**
+ * BlockTrackerPollingControllerMixin
+ *
+ * @param Base - The base class to mix onto.
+ * @returns The composed class.
+ */
+function StaticIntervalPollingControllerMixin<TBase extends Constructor>(
+  Base: TBase,
+) {
+  abstract class StaticIntervalPollingController extends PollingControllerBaseMixin(
+    Base,
+  ) {
+    readonly #intervalIds: Record<PollingTokenSetId, NodeJS.Timeout> = {};
+
+    constructor() {
+      super();
+      this.setPollingStrategy({
```

Comment:
is this main reason for the PollingStrategy object that we have different method names for starting and stopping polling between interval and block tracker polling?

Do these method names need to be so specific? I feel that the caller of these methods isn't concerned with the underlying driver of each executePoll call. Would `startByNetworkClientId` and `stopByPollingToken` not work? or even the old names?